### ISSUE TITLE
Issue pulling csi-snapshotter

### ISSUE THREAD

**Issue by andrewward** [DrFaust92/terraform-kubernetes-ebs-csi-driver#2]
Looks like you're just missing the `v` in the tag for the snapshotter image:

https://github.com/DrFaust92/terraform-kubernetes-ebs-csi-driver/blob/fe05183c4e86ada43d7f72da7ace98e5efc84cc3/locals.tf#L13

```
  Normal   BackOff    5m30s (x2 over 5m31s)  kubelet, ip-10-19-28-104.us-west-2.compute.internal  Back-off pulling image "quay.io/k8scsi/csi-snapshotter:2.1.1"
  Normal   Pulling    5m17s (x2 over 5m32s)  kubelet, ip-10-19-28-104.us-west-2.compute.internal  Pulling image "quay.io/k8scsi/csi-snapshotter:2.1.1"
  Warning  Failed     5m16s (x2 over 5m31s)  kubelet, ip-10-19-28-104.us-west-2.compute.internal  Error: ErrImagePull
  Warning  Failed     82s (x17 over 5m31s)   kubelet, ip-10-19-28-104.us-west-2.compute.internal  Error: ImagePullBackOff
```

Tangentially related: When I ran terraform after the previous failure get:

```
# module.cluster.module.csi_driver.kubernetes_deployment.ebs_csi_controller is tainted, so must be replaced
...

Error: Failed to create deployment: object is being deleted: deployments.apps "ebs-csi-controller" already exists

  on .terraform/modules/cluster.csi_driver/controller.tf line 1, in resource "kubernetes_deployment" "ebs_csi_controller":
   1: resource "kubernetes_deployment" "ebs_csi_controller" {
```

A subsequent run(with snapshotting disabled) succeeded -- maybe there's a way to block until the deployment is destroyed before continuing with the create.

**Comment by DrFaust92** [comment]
Thanks for catching that, i didnt finish check all permutations of the module yet.
Fixed and release v1.1.1 with the fix.

**Comment by DrFaust92** [comment]
regarding deletion, it seems that the k8s provider does wait for delete to finish as the call to the api is a-sync.
